src/uct/api/uct_def.h

```diff
@@ -111,6 +117,13 @@ typedef int                      uct_worker_cb_id_t;
     |<---- stride ----->|<---- stride ----->|
    @endverbatim
  *
+ * In order to describe an interleaved memory layout, each structure contains
```

Comment:
no, because count tells you how many items. For example: interleaving "A_A_A_A_" and "B_B_B_B_" (count is 4 and stride is 2) can be ABABABAB for 1:1 ratio, or AABBAABB for 2:2 ratio, or ABBABB for 1:2 ratio.